src/components/IncidentReportTracker/IncidentReportTracker.tsx

```diff
@@ -73,27 +73,27 @@ export class IncidentReportTracker extends React.PureComponent<IncidentReportTra
             delete this.active_incident_reports[report.id];
         } else {
             report.unclaim = () => {
-                post("moderation/incident/" + report.id, {"id": report.id, "action": "unclaim"})
```

Comment:
Following up.. I'm not sure I'm a fan on this %% change, % style subs have always been a bit more error prone and not as clear as just inline variable substitution, in my opinion